app/uk/gov/hmrc/pillar2/controllers/RegistrationCacheController.scala

```diff
@@ -68,11 +66,9 @@ class RegistrationCacheController @Inject() (
   }
 
   def lastUpdated(id: String): Action[AnyContent] = authenticate.async { implicit request =>
-    implicit val hc: HeaderCarrier = HeaderCarrierConverter.fromRequestAndSession(request, request.session)
-
-    logger.debug(s"[Session ID: ${SessionIdHelper.sessionId(hc)}] - controllers.RegistrationCacheController.lastUpdated: Authorised Request " + id)
+    logger.debug(s"- controllers.RegistrationCacheController.lastUpdated: Authorised Request " + id)
     repository.getLastUpdated(id).map { response =>
-      logger.debug(s"[Session ID: ${SessionIdHelper.sessionId(hc)}] - controllers.RegistrationCacheController.lastUpdated: Response " + response)
+      logger.debug(s"- controllers.RegistrationCacheController.lastUpdated: Response " + response)
       response.map { date =>
         Ok(Json.toJson(date)(jodaDateTimeNumberWrites))
       } getOrElse NotFound
```

Comment:
test for NotFound ?

Reply:
There already is a test for that  "return 404 when the data doesn't exist" [here](https://github.com/hmrc/pillar2/blob/e5b059bc8d6bfae6bebecd9cb566b8b971d1e1fb/test/uk/gov/hmrc/pillar2/controllers/RegistrationCacheControllerSpec.scala#L129)